README.md

```diff
@@ -305,6 +305,7 @@ Passing in 0 disables exponential backoff completely. In that case, the client w
 
 ### v82 -> v83
 - signature of `$seatsioClient->events->markAsForSale()` and `$seatsioClient->events->markAsNotForSale()` changed. Added new parameter `$areaPlaces`, between `$objects` and `$categories`.
+- added `$seatsioClient->charts->listCategories(string $chartKey)` to retrieve an array of `Category` instances for a specified chart.
```

Comment:
I'm fine with adding non-breaking changes like this one. We'll need to do that consistently then though - until now we only mentioned breaking stuff.